content/en/docs/cluster-administration/cluster-status-monitoring.md

```diff
@@ -0,0 +1,166 @@
+|Raft Proposals| - `Proposal Commit Rate`: Record the rate of consensus proposals committed. If the cluster is healthy, this indicator should increase over time. Several healthy members of the ETCD cluster may have different general proposals at the same time. The continuous large lag between a single member and its leader indicates that the member is slow or unhealthy. <br> - `Proposed Apply Rate`: Record the total rate of consensus proposals applied. The ETCD server applies each committed proposal asynchronously. The difference between the proposal commit rate and the proposal apply rate should usually be small (only a few thousand even under high load). If the difference between them continues to rise, it indicates that the ETCD server is overloaded. This can happen when using large-scale queries such as heavy range queries or large txn operations. <br> - `Proposal Failure Rate`: The total rate of failed proposals, usually related to two issues, temporary failures related to leader election or longer downtime due to loss of arbitration in the cluster. <br> - `Proposal Pending Total`: The current number of pending proposals. An increase in pending proposals indicates high client load or members unable to submit proposals. <br> Currently, the data displayed on the interface is the average size of the ETCD member indicators. For more information about indicators, see [etcd Server](https://etcd.io/docs/v3.3.12/metrics/#server).
+
+![ETCD Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/etcd-monitoring.png)
+
+## APIServer Monitoring
+
+[API Server](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) As the hub for the interaction of all components in the Kubernetes cluster, the following table lists the main indicators monitored by the APIServer.
+
+|Monitoring indicators|Description|
+|---|---|
+|Request Latency|Classified statistics by HTTP request method, the latency of resource request response in milliseconds|
+|Request Per Second|The number of requests accepted by kube-apiserver per second|
+
+![APIServer Monitoring](/images/docs/cluster-administration/cluster-status-monitoring/apiserver-monitoring.png)
+
+## Scheduler Monitoring
+
+[Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) Scheduler is the Kubernetes API of monitor the newly created Pod, and determine which nodes the newly created Pod should run on. It makes this decision based on available data, including the availability of collected resources and the resource requirements of the Pod. Monitoring data for scheduling delays ensure that you can see any delays faced by the scheduler.
```

Comment:
->
[Scheduler](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-scheduler/) monitors the Kubernetes API of newly created pods and determines which nodes these new pods run on. It makes this decision based on available data, including the availability of collected resources and the resource requirements of the Pod. Monitoring data for scheduling delays ensures that you can see any delays facing the scheduler.